src/GitVersionTask/GitVersionTaskBase.cs

```diff
@@ -1,72 +1,141 @@
 namespace GitVersionTask
 {
-    using System;
     using GitVersion;
     using GitVersion.Helpers;
+    using System;
+    using System.IO;
 
-    using Microsoft.Build.Framework;
-    using Microsoft.Build.Utilities;
-
-    public abstract class GitVersionTaskBase : Task
+    public static class GitVersionTaskBase
```

Comment:
Perhaps this should be called something different from `GitVersionTaskBase` now that it's not an `abstract` base class anymore?